opportunity_job_api/src/apps/jobs/models/job_offer.py

```diff
@@ -18,6 +19,7 @@ class Meta:
     job_description = models.TextField(
         verbose_name=_("job description"),
     )
+    employer = models.ForeignKey(EmployerAccount, null=True, on_delete=models.CASCADE)
```

Comment:
Bolje koristi string import (pogledaj neki drugi ForeignKey u kodu i stavi `to=` ispred naziva modela.
Formatiraj da svaki argument bude u svom redu.

Takodje, dole kod `application_deadline` fali `verbose_name`.
Taj argument treba da ima svako polje u svakom modelu.
To vazi i za ovo polje `employer`

Reply:
Ispravljeno.